scripts/jenkins/ed.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
maybe explain why this script is required, so we in the future or someone new can understand this hack